test/support/my_app_web_impl.ex

```diff
@@ -75,7 +75,11 @@ end
 
 defimpl SEO.Twitter.Build, for: MyApp.Article do
   def build(article, _conn) do
-    SEO.Twitter.build(description: article.description, title: article.title)
+    SEO.Twitter.build(
+      description: article.description,
+      title: article.title,
+      card: :summary
```

Comment:
I added this to satisfy the Twitter card assertion in `test "renders everything from item"` in `SEOTest#L24`

```elixir
assert result =~ ~s|<meta name="twitter:card" content="summary">|
```

Previously, this test was successful because the summary value was set by default.